### DISCUSSION TITLE
Gemini X settings profile associted with model ID

### DISCUSSION THREAD

**Discussion by psychotik2k3** [ExpressLRS/ExpressLRS#3068] [Ideas]
Now that gemini X is out there is one flaw that breakds the charm of ELRS: switching drone/airplane.

i mean i do have an 8 inch LR drone with gemini X module i configured on dual 2.4ghz and sub ghz, 
another of my quad is on classic elrs 2.4ghz no redudancy .
(got lot of other quads/planes but the idea is the same)

when i switch i HAVE TO open the LUA script to change the settings accordingly to the quad i want to fly. i have to remember what do i have as config on this quad etc etc...

the idea would be to be able to store the settings of gemini X mode and associate with the model ID, as edge tx is already using model ID to check we are linked to the correct model, the "only" thing would be to store the settings associated with the model ID

**Comment by pkendall64** [comment]
Use different models and model numbers for the receiver. The settings are stored per model Id already.
see https://www.expresslrs.org/software/model-config-match/?h=model#model-configuration
